packages/amplify-cli/CHANGELOG.md

```diff
@@ -3,6 +3,710 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 8.0.0 (2022-04-07)
```

Comment:
This changelog is wrong. I'm guessing this has to do with the package name change. I think this will mess up the unified changelog we generate too.